### PR TITLE
Add very simple in-browser tests

### PR DESCRIPTION
**Pull request by hobinjk**

### CODE REVIEW

**Comment by hobinjk** [comment]
Here's the current functionality (waiting on SauceLabs invite to get this to run successfully in Travis): https://imgur.com/a/BXVXN

These step screenshots would be useful for visual diffs (https://twolfson.com/2014-02-25-visual-regression-testing-in-travis-ci)

I also want to start verifying the API requests (and proper effects) that interaction with the UI generates

File: src/test/browser/load-test.js [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+  desiredCapabilities: {
+    browserName: 'firefox',
+    acceptInsecureCerts: true,
+    'moz:firefoxOptions': {
+      args: ['-headless'],
+    },
+  },
+};
+
+// 1. Choose a secure web address for your gateway
```
Comment (mrstegeman) [inline src/test/browser/load-test.js]: This comment isn't really accurate, as the test doesn't set up a domain.

**Comment by mrstegeman** [comment]
If I close Firefox, the tests will run. Still failing for me, but Travis CI doesn't seem to have issues.

```
Summary of all failing tests
 FAIL  src/test/browser/load-test.js (14.45s)
  ● basic browser tests › creates a user

    An element could not be located on the page using the given search parameters.

      at new RuntimeError (node_modules/webdriverio/build/lib/utils/ErrorHandler.js:144:12)
      at Object.<anonymous> (node_modules/webdriverio/build/lib/commands/click.js:17:19)
      at Object.exec (node_modules/webdriverio/build/lib/helpers/safeExecute.js:28:24)
      at Object.resolve (node_modules/webdriverio/build/lib/webdriverio.js:191:29)
      at node_modules/webdriverio/build/lib/webdriverio.js:540:32
      at _fulfilled (node_modules/q/q.js:854:54)
      at self.promiseDispatch.done (node_modules/q/q.js:883:30)
      at Promise.promise.promiseDispatch (node_modules/q/q.js:816:13)
      at node_modules/q/q.js:624:44
      at runSingle (node_modules/q/q.js:137:13)
      at flush (node_modules/q/q.js:125:13)
      at _combinedTickCallback (internal/process/next_tick.js:131:7)
      at process._tickDomainCallback (internal/process/next_tick.js:218:9)

  ● basic browser tests › creates a user

    ENOENT: no such file or directory, lstat '/Users/mstegeman/workspace/gateway/src/addons-test/virtual-things-adapter'

      at Object.fs.lstatSync (fs.js:941:11)
      at Object.lstatSync (node_modules/graceful-fs/polyfills.js:297:22)
      at AddonManager.uninstallAddon (src/addon-manager.js:1023:12)
          at <anonymous>
```

Any idea why that second one keeps occurring?

**Comment by hobinjk** [comment]
The second one is the test trying to clean up after itself which I have as an unconditional step. It could be wrapped in a try/catch though. The first failure happened to me when it was complaining about another gateway running locally

**Comment by mrstegeman** [comment]
@hobinjk After installing stable Firefox, instead of Nightly, I finally got a successful run. Merge away.

**Comment by hobinjk** [comment]
🎉

**Comment by mrstegeman** [comment]
@hobinjk Could you add the new top-level `.png` files to `.gitignore`, please?

**Comment by hobinjk** [comment]
Yes, they'll all be in their own gitignored directory as is proper

**Pull request merged**